controllers/charity-controllers.js

```diff
@@ -66,3 +68,13 @@ exports.sendCharityRequirement = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCharityRequest = (req, res, next) => {
+  const { request_id } = req.params;
+
+  Promise.all([checkCharityRequestExists(request_id), removeCharityRequest(request_id)])
```

Comment:
I'm not sure this would work 100% of the time as if the second promise resolves before the first then it would throw a psql error not the custom error below?